donald_wade/week_02/styles2.css

```diff
@@ -0,0 +1,61 @@
+* {
+  /*border: 1px dotted red;*/
+  box-sizing: border-box;
+}
+body {
+  height: 100vh;
+  width: 100vw;
+}
+
+.box {
+  font-size: 3em;
+  padding: 5%;
+  width: 50vw;
+  height: 50vh;
+  transition: all 0.8s ease;
+}
+
+.one {
+  background-color: green;
+  display: inline;
+  float: left;
+  transition: all 0.5s ease;
+
+}
+.box:hover {
+  background-color: white;
+
+}
+.two {
+  background-color: yellow;
+  float: right;
```

Comment:
try to avoid float. It ll cause lot's of problems. Use relative and absolute positioning to achieve same result